plugins/woocommerce/tests/e2e-pw/tests/merchant/page-loads.spec.js

```diff
@@ -70,20 +70,20 @@ const wcPages = [
 			{
 				name: 'Categories',
 				heading: 'Product categories',
-				element: '.row-title',
-				text: 'Uncategorized',
+				element: '#submit',
+				text: 'Add new category',
 			},
 			{
 				name: 'Tags',
 				heading: 'Product tags',
-				element: '.no-items > td',
-				text: 'No tags found',
+				element: '#submit',
+				text: 'Add new tag',
 			},
 			{
 				name: 'Attributes',
 				heading: 'Attributes',
-				element: '.alternate > td',
-				text: 'No attributes currently exist.',
+				element: '#submit',
+				text: 'Add attribute',
```

Comment:
I changed these to be elements that are always on the page because if these tests ran while tests were running for categories/attributes, there may not be an empty page.